clients/iscmove/iscmoveclient/client_anchor_ptb.go

```diff
@@ -262,6 +295,35 @@ func PTBReceiveRequestsAndTransition(
 				},
 			)
 		}
+		for _, id := range slices.SortedFunc(
+			slices.Values(maps.Keys(assetsBag.Objects)),
+			func(a iotago.ObjectID, b iotago.ObjectID) int {
+				return bytes.Compare(a[:], b[:])
+			},
+		) {
+			obj := ptb.Command(
+				iotago.Command{
+					MoveCall: &iotago.ProgrammableMoveCall{
+						Package:       &packageID,
+						Module:        iscmove.AssetsBagModuleName,
+						Function:      "take_asset",
+						TypeArguments: []iotago.TypeTag{assetsBag.Objects[id].TypeTag()},
+						Arguments:     []iotago.Argument{argAssetsBag, ptb.MustPure(id)},
```

Comment:
If the object is deposit and withdraw in the same PTB, then ` ptb.MustPure(id)` may cause error, because the same `id` will show up more than once.

Reply:
Ok, should I replace it with ptb.MustForceSeparatePure ?